prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -347,23 +361,20 @@ elements:
     # column BH
     - name: pregnant
       csvFields: [ { name: Pregnancy Status } ]
+      altValues:
+          - display: Pregnant
+            code: Y
+          - display: Not Pregnant
+            code: N
+          - display: Unknown
+            code: U
 
     # column BI
     # nonstandard
     - name: student_teacher_other
       type: CODE
+      valueSet: fl/student_teacher_other
       csvFields: [{ name: "Is the patient a student, teacher, or other faculty member" }]
-      altValues:
```

Comment:
Change to patient_role, and make that use a covid-19.valueset valueSet of these values.
Then write a one-off altValues here to cover the Other situation.

Reply:
`patient_role` doesn't exist yet in the standard schema, and LOINC are all listed as TBD. This is a little different than occupation, though there's some overlap. That said `patient_occupation` is also not in the schema yet. We should probably sit with SimpleReport and discuss how they're capturing that information. I'd like to leave this as is until we start adding LOINC values or build in the occupation field.